src/definitions/modules/calendar.js

```diff
@@ -298,7 +298,6 @@ $.fn.calendar = function(parameters) {
                   if(settings.showWeekNumbers) {
                       cell = $('<th/>').appendTo(row);
                       cell.text(settings.text.weekNo);
-                      cell.addClass(className.disabledCell);
```

Comment:
Entirely removing the disabled class for the header will result in always displaying the Week-Number Text in the same prominent style as the weekdays which might be considered a breaking change (as the user cannot change it anymoire now)
I suggest to either have a separate option for the headerCell Class or use the same className.weekCell option for this to avoid being a breaking change and more ensure consistency

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/18379884/54121888-e20f0c00-43fb-11e9-974f-2244780b9e41.png)|![image](https://user-images.githubusercontent.com/18379884/54121838-c3a91080-43fb-11e9-8128-f262844bf786.png)|

Reply:
Updated PR: The week cell class is applied to header and normal row cell.

![image](https://user-images.githubusercontent.com/3204107/54124178-48029000-440a-11e9-8efa-4f3b043f06e4.png)